generators/bdk/index.js

```diff
@@ -19,6 +20,15 @@ module.exports = class extends Generator {
     this.log('Welcome to Symphony Generator '.gray + `v${packageJson.version}`.yellow);
     this.log('Application files will be generated in folder: '.gray + `${this.destinationRoot()}`.yellow);
     this.log('______________________________________________________________________________________________________'.yellow);
+
+    try {
+      const arrayOfFiles = fs.readdirSync(this.destinationRoot());
+      if (arrayOfFiles.length > 0) {
+        this.log(`(!) Folder ${this.destinationRoot()} is not empty. Are you sure you want to continue?`.red);
+      }
+    } catch(e) {
```

Comment:
the catch is here is the directory does not exist yet? could we check for existence instead of relying on the exception maybe?

Reply:
Actually when the folder does not exist, it is created beforehand and when we arrive at this level (bdk/index), an empty folder is already created. The catch is only for fs.readdirSync potential errors